pkgs/applications/kde/srcs.nix

```diff
@@ -4,1867 +4,1875 @@
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/blinken-22.08.3.tar.xz";
-      sha256 = "1avdphqp1y528lczb6bpxvdzaq9navrg4vxiqvfkvgkymhd36n2k";
-      name = "blinken-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/blinken-22.12.0.tar.xz";
+      sha256 = "0ll62mvbfw7ywh5y9j6l2id75a8gyqspynnhwy1cxn0jm8bpixyl";
+      name = "blinken-22.12.0.tar.xz";
     };
   };
   bomber = {
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/bomber-22.08.3.tar.xz";
-      sha256 = "1fm5fisxa5q2shc3154g4cfg271jz97411wfjcdpgry6mx96qs7k";
-      name = "bomber-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/bomber-22.12.0.tar.xz";
+      sha256 = "06dhaw9kvhqj7a6cbhaj5q39w84fapzilfs5amkamib470zmwx4f";
+      name = "bomber-22.12.0.tar.xz";
     };
   };
   bovo = {
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/bovo-22.08.3.tar.xz";
-      sha256 = "060gc854krafa4ichknj5naic0ingiviacbz1lmdg2fp68mggd50";
-      name = "bovo-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/bovo-22.12.0.tar.xz";
+      sha256 = "1s3drsjrbxyfqk140mxdg5x8zdcgw3brnpb8nsnmdp6jpphr9p74";
+      name = "bovo-22.12.0.tar.xz";
     };
   };
   calendarsupport = {
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/calendarsupport-22.08.3.tar.xz";
-      sha256 = "1lmysb5g0xsaj5n75ays9hmjgcg71apg7fw3fg93rdh9hzn9in0x";
-      name = "calendarsupport-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/calendarsupport-22.12.0.tar.xz";
+      sha256 = "15sgvswx3jwh9162jax0m21qvzcjp0m1fzpbgr0xjqy9gkirg97q";
+      name = "calendarsupport-22.12.0.tar.xz";
     };
   };
   cantor = {
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/cantor-22.08.3.tar.xz";
-      sha256 = "017zzxl60ycjc24i9zv7mjkxpdx4fa8kznxmfdhc601sdh1n0pja";
-      name = "cantor-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/cantor-22.12.0.tar.xz";
+      sha256 = "0bzxvc6yzfgf8ci9ipw144yack4r53xg3y6qkg2nbs0g4364yayw";
+      name = "cantor-22.12.0.tar.xz";
     };
   };
   cervisia = {
-    version = "22.08.3";
+    version = "22.12.0";
     src = fetchurl {
-      url = "${mirror}/stable/release-service/22.08.3/src/cervisia-22.08.3.tar.xz";
-      sha256 = "1q668vd237wf7nd5sjygnkbygbj6qwbc9j0l3pgpbd51r220jh04";
-      name = "cervisia-22.08.3.tar.xz";
+      url = "${mirror}/stable/release-service/22.12.0/src/cervisia-22.12.0.tar.xz";
+      sha256 = "16ravvkkj2n4ca0mv44h9bmywls6qmd12y09rbqi612p5baxcm4g";
+      name = "cervisia-22.12.0.tar.xz";
+    };
+  };
+  colord-kde = {
```

Comment:
New package, I wonder where this is used.

Reply:
sounds like a new systemd component

Reply:
It is an interface to color to manage monitor color profiles

Reply:
We already have `colord-kde` in nixpkgs, but as it is now a part of KDE gear, we should drop the "standalone" version and use this instead.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/colord-kde/default.nix

Reply:
Moved `colord-kde` now